components/AttributeList/AttributeListItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Chip, TableCell, TableRow } from '@mui/material'
+import CircleIcon from '@mui/icons-material/Circle'
+import { AttributeResponse } from 'utils/types'
+import AttributeListItemKebab from '../AttributeListItemKebab'
+
+interface AttributeListItemProps {
+  attribute: AttributeResponse
+}
+
+export default function AttributeListItem({
+  attribute,
+}: AttributeListItemProps) {
+  return (
+    <TableRow>
+      <TableCell>{attribute.name}</TableCell>
+      <TableCell>
+        {typeof attribute.possibleValues === 'object'
+          ? attribute.possibleValues.map((possibleValue, index) => (
+              <Chip
+                size="small"
+                label={possibleValue}
+                key={`${possibleValue}-${index}`}
+              />
+            ))
+          : attribute.possibleValues}
```

Comment:
Can you capitalize the first letter of this? So it'd display 'Number' instead of 'number'